executor/runtime/docker/docker.go

```diff
@@ -108,6 +108,10 @@ var NoEntrypointError = &runtimeTypes.BadEntryPointError{Reason: errors.New("Ima
 // applications shall tolerate the presence of such names.
 var environmentVariableKeyRegexp = regexp.MustCompile("^[A-Za-z_][A-Za-z0-9_]*$")
 
+// possibleSystemdPaths is a list of paths that represent commands that end up running systemd.
+// We need this so that we can know if we should `exec` into them or not (TINI_HANDOFF)
+var possibleSystemdPaths = []string{"/sbin/init", "/nflx/init", "/lib/systemd/systemd"}
```

Comment:
Should these be a configurable?

Reply:
Eh. I don't think so. @bmoyles assures me that those are the only 3.
If we get more in the wild, I'll roll back and add it in.